docs/troubleshooting.md

```diff
@@ -2,9 +2,60 @@
 
 This section contains commands and tips to troubleshoot Fleet.
 
-## Check status of fleet controller
+## **How Do I...**
 
-You can check the status of the Fleet controller pods by running the commands below:
+
+### Fetch the log from `fleet-controller`?
+
+In the local management cluster where the `fleet-controller` is deployed, run the following command with your specific `fleet-controller` pod name filled in:
+
+```
+kubectl logs -f fleet-controller-pod-name -n cattle-fleet-system
+```
+
+### Fetch the log from the `fleet-agent`?
+
+Go to each downstream cluster and run the following command for the local cluster with your specific `fleet-agent` pod name filled in:
+
+```
+# Downstream cluster
+kubectl logs -f fleet-agent-pod-name -n cattle-fleet-system
+# Local cluster
+kubectl logs -f fleet-agent-pod-name -n cattle-local-fleet-system
+```
+
+### Fetch detailed error logs from `GitRepos` and `Bundles`?
+
+Normally, errors should appear in the Rancher UI. However, if there is not enough information displayed about the error there, you can research further by trying one or more of the following as needed:
+
+- For more information about the bundle, click on `bundle`, and the YAML mode will be enabled. 
+- For more information about the GitRepo, click on `GitRepo`, then click on `View Yaml` in the upper right of the screen. After viewing the YAML, check `status.conditions`; a detailed error message should be displayed here.
+- Check the `fleet-controller` for synching errors.
+- Check the `fleet-agent` log in the downstream cluster if you encounter issues when deploying the bundle.
+
+### Check a chart rendering error in `Kustomize`?
+
+Check the `fleet-controller` logs and the `fleet-agent` log.
+
+### Check errors about watching or checking out the `GitRepo`, or about the downloaded Helm repo in `fleet.yaml`?
+
+Check the `gitjob-controller` logs using the following command with your specific `gitjob` pod name filled in:
+
+```
+kubectl logs -f gitjob-pod-name -n cattle-fleet-system
+```
+
+Note that there are two containers inside the pod: the `step-git-source` container that clones the git repo, and the `fleet` container that applies bundles based on the git repo. 
+
+The pods will usually have images named `rancher/tekton-utils` with the `gitRepo` name as a prefix. Check the logs for these Kubernetes job pods in the local management cluster as follows, filling in your specific `gitRepoName` pod name:
```

Comment:
```suggestion
The pods will usually have images named `rancher/tekton-utils` with the `gitRepo` name as a prefix. Check the logs for these Kubernetes job pods in the local management cluster as follows, filling in your specific `gitRepoName` pod name and namespace:
```

The namespace needs to be filled to, since it goes with the same namespace where users create their gitRepo.

Reply:
Amended per feedback.